src/nft/hooks/useNFTSelect.ts

```diff
@@ -25,7 +25,7 @@ export const useNFTSelect = create<SelectNFTState>()(
       selectNFT: (nft) =>
         set(({ selectedNFTs }) => {
           if (selectedNFTs.length === 0) return { selectedNFTs: [nft] }
```

Comment:
This bracket pattern should be linted against. If there is an `else`, it should use multi-line brackets. Only a one-liner guard should be allowed to omit the brackets (eg `if (condition) return earlyReturnValue`).

Reply:
I am going to write it down as a follow-up to this PR. This will more than likely affect other parts of the codebase too. Overall I am in favor.